static/docs/reference/generated/kubectl/kubectl-commands.html

```diff
@@ -3017,7 +3017,7 @@ <h3 id="flags">Flags</h3>
 <td>cascade</td>
 <td></td>
 <td>background</td>
-<td>Must be &quot;background&quot;, &quot;orphan&quot;, or &quot;foreground&quot;. Selects the deletion cascading strategy for the dependents (e.g. Pods created by a ReplicationController). Defaults to background. </td>
+<td>Must be &quot;background&quot;, &quot;orphan&quot;, &quot;foreground&quot; or false. Selects the deletion cascading strategy for the dependents (e.g. Pods created by a ReplicationController). Defaults to background. </td>
```

Comment:
This change removes the Oxford comma. It does not clarify whether false and orphan differ.  The changed line references the deprecated ReplicationController.

Moreover I do not get the meanimg of foreground and background values